core/src/main/scala/core/database/ComposeWithCompletion.scala

```diff
@@ -2,56 +2,47 @@ package com.zengularity.querymonad.core.database
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.higherKinds
-import scala.util.Try
 
 /**
- * Heavily inspired from work done by @cchantep in Acolyte (see acolyte.reactivemongo.ComposeWithCompletion)
- */
+  * Heavily inspired from work done by @cchantep in Acolyte (see acolyte.reactivemongo.ComposeWithCompletion)
```

Comment:
Indent issue

Reply:
Yes, I have an issue with scalafmt which doesn't format properly `scaladoc`. I'm working on an PR to update scalafmt settings.